### ISSUE TITLE
Support spaces in file url

### ISSUE THREAD

**Issue by martinjuhasz** [evi-snowm/cert-downloader#2]
I tried verifying an ipa file that contains a space in its filepath:

```javascript
var certDl = new (require('cert-downloader'));
  certDl.verify('storage/Ipa Help.ipa', function(error, output) {
    if(error){
      return callback(error);
    }
    callback(null, plist.parse(output));
  });
```

This fails because the filepath string is directly added to the openssl command without escaping.
changing the command line from
```
openssl smime -in %s -inform der -verify -CAfile %s'
```
to 
```
openssl smime -in "%s" -inform der -verify -CAfile %s'
```
would prevent this.

What do you think about my issue?

**Comment by evi-snowm** [comment]
Whoops... totally forgot to build a new release for that issue; this was fixed on master 5 months ago.

Sorry about that, I will double-check the code and wrap the fix into a new version asap. Thank you for your report!

**Comment by evi-snowm** [comment]
Just published version 0.2.1, this should address your issue.

**Comment by martinjuhasz** [comment]
Thank you, works fine :)
